modules/standard/BigInteger.chpl

```diff
@@ -2595,6 +2628,24 @@ When ``numer/denom`` does not produce an integer, this method may produce incorr
       return false;
   }
 
+  /*
+    Return ``true`` if ``this`` is congruent to ``con`` modulo ``mod``.
+    ``this`` is congruent to ``con % mod`` if there exists an integer ``q``
```

Comment:
The first line says "`con` modulo `mod`" and the second says `con % mod`. Maybe pick one or the other to use in both places?

Reply:
Updated, thoughts?

Reply:
I'm happy with that consistency